GameServerLib/Logic/GameObjects/Unit.cs

```diff
@@ -63,7 +63,7 @@ public class Unit : GameObject
         internal const int EXP_RANGE = 1400;
         internal const long UPDATE_TIME = 500;
 
-        protected Stats stats;
+        protected Stats Stats;
```

Comment:
Since you changed the case, please make this a property as well (our style guide says that variables are lowerCase and properties PascalCase)
```C#
protected Stats Stats { get; set; }
```